src/components/Stats.tsx

```diff
@@ -10,50 +10,63 @@ import MonthlyDataTab from "./MonthlyDataTab";
 export interface StatsProps {
   libraries?: LibraryData[];
 }
+export interface StatsState {
+  panelOpen: boolean;
+}
 
 /** The Aggregate Data panel. */
-export default class Stats extends React.Component<StatsProps, {}> {
+export default class Stats extends React.Component<StatsProps, StatsState> {
   constructor(props: StatsProps) {
     super(props);
+    this.state = { panelOpen: false };
```

Comment:
Same here as the other components, maybe this should be `this.props.panelOpen` instead of `false`.

Reply:
Again, this is fine as is!